src/shogun/neuralnets/NeuralNetwork.h

```diff
@@ -442,6 +442,18 @@ friend class CDeepBeliefNetwork;
 	 * default value is -1
 	 */
 	float64_t gd_error_damping_coeff;
+
+	/** Used in RmsProp stochastic optimization 
+	* RmsProp scales each gradient down by its running average.
+	* It then updates the parameters acc. to the learning_rate.
+	*     scale=decay_factor*scale+(1-decay_factor)*gradient*gradient
```

Comment:
Can we get these equations with Latex? See for instance the doc of check_gradients in this file.